ios/application_spriteKit_game.nic/Makefile

```diff
@@ -0,0 +1,14 @@
+include @@THEOS@@/makefiles/common.mk
+
+APPLICATION_NAME = @@PROJECTNAME@@
+$(APPLICATION_NAME)_FILES = $(wildcard *.m)
```

Comment:
Can you change these `$(APPLICATION_NAME)` to `@@PROJECTNAME@@` please?

Reply:
It is way more easier to use `$(APPLICATION_NAME)`
Because if someone wants to rename their project the only thing they need to change is the `APPLICATION_NAME` Variable 

A lot of developers recommended this approach

I can change it but I thought this is easier to deal with when changing names

Reply:
If you want your makefile to only work in a subset of use cases and cause very strange errors that confuse people in other cases, sure.  I’m pretty sure you will need to write proper code if you want it to be merged into mainline theos though.

Reply:
So I need to change it to `@@PROJECTNAME@@` ?